docs/sources/dashboards/use-dashboards/index.md

```diff
@@ -272,3 +277,30 @@ Selecting the **Auto** interval schedules a refresh based on the query time rang
 ### Control the time range using a URL
 
 {{< docs/shared lookup="dashboards/time-range-URLs.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+
+## Filter dashboard data
+
+Once you've [added an ad hoc filter](ref:add-ad-hoc-filters) in the dashboard settings, you can create key/value filter pairs on the dashboard.
+These filters are applied to all metric queries that use the specified data source and to all panels on the dashboard.
+
+To filter dashboard data, follow these steps:
+
+1. On the dashboard, click in the filter field.
+1. Select a label, operator, and value.
+1. Repeat this process as needed until you have all the filters you need.
+
+   ![Ad hoc filters](/media/docs/grafana/dashboards/screenshot-adhoc-filters-v11.3.png)
+
```

Comment:
I'm missing steps on how to add multiple values to a filter, but I can't get it to work. Help!